app/Http/Controllers/LogoutController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class LogoutController extends Controller
+{
+    public function logout(Request $request) {
+        $session =  session('username');
```

Comment:
Is this needed?

Reply:
Lol you wrote it. It's your function. But no. It isn't